### PR TITLE
Remove aws account id and update readme

### PR DESCRIPTION
**Pull request by Stuart-Knowles-SW**
Pass in the AWS Account id as an environment variable rather than hardcoded

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,10 @@
 - Node.js
 
 ## Get AWS credentials
-Go to [Softwire AWS Start page](https://softwire.awsapps.com/start#/) and select VFB and the appropriate role to assume. 
-Click on `Command line or programmatic access`, and copy the commands to inject your credentials in the terminal.
+1. Go to [Lambeth Foodbank AWS Start page](https://d-9c6760eaca.awsapps.com/start/#/?tab=accounts) and login
```
Comment (stonelink) [inline README.md:7]: Does the repo need to have a link to the AWS account? 
The foodbank will need to maintain some swiki-style technical documentation that links to accounts etc.

File: README.md [hunk 2]
```diff
@@ -4,8 +4,10 @@
 - Node.js
 
 ## Get AWS credentials
-Go to [Softwire AWS Start page](https://softwire.awsapps.com/start#/) and select VFB and the appropriate role to assume. 
-Click on `Command line or programmatic access`, and copy the commands to inject your credentials in the terminal.
+1. Go to [Lambeth Foodbank AWS Start page](https://d-9c6760eaca.awsapps.com/start/#/?tab=accounts) and login 
+2. Click on "Access Keys" under the LambethFoodBank account, and follow the instructions to inject your credentials in the terminal.
+3. Additionally, set the environment variable `AWS_ACCOUNT_ID` to the id for the same AWS account
+4. To check this has worked, run `aws s3 ls`. You will probably get a single line returned (something like `2024-05-16 11:13:28 cdk-hnb659fds-assets-471112736904-eu-west-2`), but as long as it doesn't error you're probably connected.
```
Comment (stonelink) [inline README.md:10]: Can the example be a bit obfuscated, please - maybe just replacing the `hnh...` bit and `4711...` by `xxxxxx`?

**Review by stonelink — COMMENTED** [review]
Just a couple of comments about some more things that can be removed/obfuscated

**Comment by Stuart-Knowles-SW** [comment]
Both excellent points, have updated

**Review by stonelink — APPROVED** [review]
Great 👍

**Pull request merged**